easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -49,7 +49,6 @@ class EB_QuantumESPRESSO(ConfigureMake):
     def extra_options():
         """Custom easyconfig parameters for Quantum ESPRESSO."""
         extra_vars = {
-            'hybrid': [False, "Enable hybrid build (with OpenMP)", CUSTOM],
```

Comment:
Or is the case that people would want to enable hybrid without the toolchain option?